README.md

```diff
@@ -25,26 +25,87 @@ Want to get paid for your contributions to `react-seo`?
 ```bash
 npm install @americanexpress/react-seo
 ```
+
+<br />
+
+Let's start with a minimal example of basic usage:
+
 ```javascript
 import React from 'react';
 import SEO from '@americanexpress/react-seo';
 
 const MyModule = () => (
   <div>
     <SEO
-      author="John Doe"
```

Comment:
@jgolden17 how do you pass the author now?

Reply:
I haven't added a schema type of articles yet.  Will have a fast follower to add that.